internal/versions/versions.go

```diff
@@ -109,17 +129,95 @@ func (h *handler) GetRHCOSVersion(openshiftVersion string) (string, error) {
 }
 
 func (h *handler) IsOpenshiftVersionSupported(openshiftVersion string) bool {
```

Comment:
It's a little bit confusing that other function first call `GetSupportedKeyVersionFormat` and then `IsOpenshiftVersionSupported` calling the same formatting once again.
So either this function should expect a key notation 
```suggestion
func (h *handler) IsOpenshiftVersionKeySupported(key string) bool {
```
Or the other functions should call it before formating the version - without the formatted key